katello/katello-change-hostname

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/ruby
+
+require "socket"
+require 'highline/import'
+
+STDOUT.puts "Starting the hostname change..."
+
+STDOUT.puts("\nA hostname change requires a manual change in Satellite. \n
+            Please log into UI and rename the default capsule (under infrastructure->capsules) 
+            to the new hostname using both the name and url fields.")
+
+unless HighLine.agree('Have you made this necessary change and want to proceed with the hostname change? [yes/no]')
+  STDOUT.puts "Hostname change aborted, no changes have been made to your system"
+  exit
+end
+
+if ARGV[0] && ARGV.count == 1
+  NEW_HOSTNAME = ARGV[0]
+else
+  STDOUT.puts "USAGE: katello-change-hostname <new-hostname>"
+  exit
+end
+
+# Get the hostname from your system
+OLD_HOSTNAME = Socket.gethostname
```

Comment:
should we check here that OLD_HOSTNAME != the new hostname?  In case the user thought they should go ahead and do that manually (which would cause future actions to fail).